lib/galaxy/util/sanitize_html.py

```diff
@@ -194,7 +194,7 @@ def convert_entityref(self, name):
 
     def output(self):
         '''Return processed HTML as a single string'''
-        return ''.join([str(p) for p in self.pieces])
+        return ''.join([unicodify(p) for p in self.pieces])
```

Comment:
[Upstream](https://github.com/kurtmckee/feedparser/blob/develop/feedparser/html.py#L205) just does:
```
return ''.join(self.pieces)
```
Not sure this will work for Galaxy, but with @mvdbeek we were discussing to try to remove/rework this file and depend on a `feedparser` wheel, if I remember well.